src/controllers/discussionController.ts

```diff
@@ -221,6 +222,29 @@ const joinDiscussion = async (
 
     discussion.participants.push(user.userId);
     await discussion.save();
+
+     const meetingTime = new Date(discussion.date).toLocaleString();
```

Comment:
you can create a new file called emailService.ts then move all the email logic into it, then you call the service here like:
emailService.sendEmail(user,discussion,meetingLink, emailType)

email: type: meeting/unsubscribed/...

Reply:
i already have this file: import { sendEmail } from "../services/mailjetService"; i refactored a little my code. In this file i only created the content for email because it can be different